### PR TITLE
new 'labels' optional parameter for create_prod_build()

### PR DESCRIPTION
**Pull request by twaugh**
This parameter provides overrides for the labels in the Dockerfile. It cannot be set in the configuration file but can be provided at the command line.

Not providing configuration file compatibility because (a) it was never documented, and (b) config values would not get merged with CLI/kwarg values.

### CODE REVIEW

**Comment by lcarva** [comment]
LGTM

File: osbs/cli/main.py [hunk 1]
```diff
@@ -263,6 +263,11 @@ def cmd_cancel_build(args, osbs):
 
 
 def cmd_build(args, osbs):
+    if args.labels:
+        labels = dict(args.labels)
+    else:
+        labels = None
```
Comment (vrutkovs) [inline osbs/cli/main.py]: Nitpick: wouldn't label var have a wrong var scope here? I guess we should set it to None first and then make a dict if ```args.labels``` is set

Comment (lcarva) [inline osbs/cli/main.py]: Python doesn't have the concept of block scoping. In this case it's function scope.

Comment (vrutkovs) [inline osbs/cli/main.py]: Thanks, these statements always confuse me after several years of Ruby

**Comment by vrutkovs** [comment]
LGTM, I tested a similar patch. Merging

**Pull request merged**